docs/resources/database_postgresql.md

```diff
@@ -0,0 +1,76 @@
+# Resource: <resource name> delphix_database_postgresql
+
+In Delphix, a databsae/source is required in the environment using which a dsource can be created.
+
+## System Requirements
+
+* Data Control Tower v14.0.1+ is required for dSource management. Lower versions are not supported.
+* This Databse PostgresSql Resource only supports POSTGRES , This resource does not support Oracle, SQL Server, or SAP ASE.
+
+## Example Usage
+
+```hcl
+# Create a postgres database/source. 
+
+resource "delphix_database_postgresql" "source" {
+  name             = "test"
+  repository_value = "REPO-1"
+  engine_value = "2"
+  environment_value = "ENV-1"
+}
+
+```
+
+## Argument Reference
+
+* `name` - (Required) The name of the new source.
+
+* `repository_value` - (Required)  The Id or Name of the Repository onto which the source will be created..
+
+* `environment_value` - (Required) The Id or Name of the environment to create the source on.
+
+* `engine_value` - (Required) The Id or Name of the engine to create the source on.
+
+* `id` - The Source object entity ID.
+
+* `database_type` - The type of this source database.
+
+* `id` - The Source object entity ID.
+
+* `database_type` - The type of this source database.
+
+* `namespace_id` - The namespace id of this source database.
+
+* `namespace_name` - The namespace name of this source database.
+
+* `is_replica` - Is this a replicated object.
+
+* `database_version` - The version of this source database.
+
+* `data_uuid` - A universal ID that uniquely identifies this source database.
+
+* `ip_address` - The IP address of the source's host.
+
+* `fqdn` - The FQDN of the source's host.
+
+* `size` - The total size of this source database, in bytes.
+
+* `jdbc_connection_string` - The JDBC connection URL for this source database.
+
+* `plugin_version` - The version of the plugin associated with this source database.
+
+* `toolkit_id` - The ID of the toolkit associated with this source database(AppData only).
+
+* `is_dsource` - Is this associated with dSource.
+
+* `repository` - The repository id for this source.
+
+* `recovery_model` - Recovery model of the source database (MSSql Only).
+
+* `mssql_source_type` - The type of this mssql source database (MSSql Only).
```

Comment:
We should remove these two MS SQL properties since it's not supported. (Or at least hide them)

Reply:
agree